flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvToRowDataConverters.java

```diff
@@ -185,48 +187,57 @@ private CsvToRowDataConverter createConverter(LogicalType type) {
         }
     }
 
-    private boolean convertToBoolean(JsonNode jsonNode) {
+    private <V> V convertStringToValue(JsonNode jsonNode, Function<String, V> function) {
```

Comment:
`createNullableConverter` should have already handled nulls (similar to json format), however, for csv, this does not work because blank fields is represented as empty text node (empty string). I'm not sure whether there is any configuration for jackson csv mapper to represent blank fields to a null node?

Reply:
There is already a config in csv format about null literal `null-literal`, I guess setting it to empty string would solve the problem?

Reply:
I try the option `null-literal` and it works. But I think there're some improvement points about csv format here. Currently when I create a csv format table with nullable fields such as table ( a int, b int). There are no additional options for the table and I can successfully insert null data to the field a and b. When I try to select results from the table, it throw exception that empty string('') cannot be parsed as int value. I think there may be two solutions for this, and we can pick one:
1. Throw an exception and tell users to configure `null-literal` for their csv table if they try to insert a null value.
2. Give the option `null-literal` a default value such as empty string('')

@libenchao What do you think of it?

Reply:
> and go for 2 finally, maybe in Flink 2.0?

Do propose it for Flink 2.0 indeed!